### DISCUSSION TITLE
is there restrictions to changing Apple's own OSX app icons?

### DISCUSSION THREAD

**Discussion by marktmcewan** [elrumo/macOS_Big_Sur_icons_replacements#1404] [Help with the website] [answered]
Hi everyone, I have been trying to change the native TV app icon on my iMac with no success.
I have been trying to swap out the default icon for this one here "Apple TV (Alt) by Mr CopyRight" but the change does not render.

I have changed many other app icons with no issues at all. I am curious if apple's own apps are locked from changes?

**Comment by nguyenkien** [comment, marked as answer]
Default app stay in read-only partition. You need mount it as read-write to change.

**Reply by marktmcewan** [reply]
Thanks Nguyen

**Comment by gugu-7** [comment]
how can you change read-write for mac os own Apps?

**Comment by TheManchineel** [comment]
You need to disable System Integrity Protection (SIP). Please note that this will decrease system security and break incremental updates (often when you update the OS, it will be redownloaded whole, i.e. lots of GB of updates each time, you'll also have to reapply these changes after the update), along with support for iOS apps on Apple silicon and a bunch of other things.

Whether it's worth it just to change an icon is up to you.

**Comment by MysterienVril** [comment]
Can you go into the applications folder and create a copy of the App and then change the icon of the copy and use a shortcut of the copy? that is what I have done with my Apple icons like Safari and Messages.

**Reply by pan-jessica** [reply]
I believe this worked with updates prior to Ventura? After updating to Ventura, I wasn't able do that to native applications--like Messages, Settings, FaceTime, etc. Though, I had my duplicated version of Safari open during my update to Ventura, and it was able to keep the duplicated version with my modified icon after the update; all the other duplicated native applications were removed after the update. Though, I am curious if anyone knows how Safari was able to get through.
